ext/session/tests/bug32330.phpt

```diff
@@ -6,7 +6,7 @@ Bug #32330 (session_destroy, "Failed to initialize storage module", custom sessi
 session.use_trans_sid=0
 session.use_cookies=1
 session.name=sid
-session.save_path=/tmp
+session.save_path={TMP}
```

Comment:
Changing this here, requires to also adjust the test expectations like @petk mentioned. Otherwise the test may fail (see https://ci.appveyor.com/project/php/php-src/builds/24580090/job/y9705h6p5o2kwlbp#L5909).

Reply:
Changed, Thanks!